components/Forms/CreateResourceForm/CreateResourceForm.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { func, arrayOf, shape, string, number } from 'prop-types';
+import { Formik, Field } from 'formik';
+import * as Yup from 'yup';
+import { createResource } from 'common/constants/api';
+import { getServerErrorMessage } from 'common/utils/api-utils';
+import { validationErrorMessages } from 'common/constants/messages';
+import Button from 'components/Buttons/Button/Button';
+import Form from 'components/Form/Form';
+import Input from 'components/Form/Input/Input';
+import Select from 'components/Form/Select/Select';
+import styles from './CreateResourceForm.module.css';
+
+// This exists only because we don't have a radio component and I wanna get this done quickly!
+// TODO: Make radio input component
+const free = { value: 'true', label: 'Free' };
+const paid = { value: 'false', label: 'Paid' };
+
+const costOptions = [free, paid];
+
+const validationSchema = Yup.object().shape({
+  name: Yup.string().required(validationErrorMessages.required).default(''),
+  url: Yup.string().required(validationErrorMessages.required).default(''),
+  category: Yup.string().required(validationErrorMessages.required).default(''),
+  languages: Yup.array().default(['']),
```

Comment:
This doesn't matter for creating, but when we do a `PUT` to update a resource, if the languages are not meant to change, the client should omit the `languages` rather than sending `[]` because the latter will clear out all existing languages.

And also, we should probably remove the quotes here, because it should be a default of `[]` rather than `['']`

Reply:
So this spot is for compatibility with the multi select component; however, in the submit handler we can translate your needs there much how we've done with cost --> free.